src/stories/Tokens/Typography/FontFamily.mdx

```diff
@@ -56,21 +56,23 @@ If you are using a custom font, you can adjust the axes of the font by using the
 For example: [Roboto Flex](https://fonts.google.com/specimen/Roboto+Flex) has the following axes.
 
 ```css
-// <uniquifier>: Use a unique and descriptive class name
-// <weight>: Use a value from 100 to 1000
-// <grade>: Use a value from -200 to 150
-// <slant>: Use a value from -10 to 0
-// <width>: Use a value from 25 to 151
-// <thick stroke>: Use a value from 27 to 175
-// <thin stroke>: Use a value from 25 to 135
-// <counter width>: Use a value from 323 to 603
-// <uppercase height>: Use a value from 528 to 760
-// <lowercase height>: Use a value from 416 to 570
-// <ascender height>: Use a value from 649 to 854
-// <descender depth>: Use a value from -305 to -98
-// <figure height>: Use a value from 560 to 788
-
-.roboto-flex-<uniquifier> {
+/*
+  <uniquifier>: Use a unique and descriptive class name
+  <weight>: Use a value from 100 to 1000
+  <grade>: Use a value from -200 to 150
+  <slant>: Use a value from -10 to 0
+  <width>: Use a value from 25 to 151
+  <thick stroke>: Use a value from 27 to 175
+  <thin stroke>: Use a value from 25 to 135
+  <counter width>: Use a value from 323 to 603
+  <uppercase height>: Use a value from 528 to 760
+  <lowercase height>: Use a value from 416 to 570
+  <ascender height>: Use a value from 649 to 854
+  <descender depth>: Use a value from -305 to -98
+  <figure height>: Use a value from 560 to 788
+*/
+
+.roboto-flex-<uniquifier > {
```

Comment:
@Jeremy-Walton Is the extra space intended here?

Reply:
Good catch. The auto formatting is adding it because it thinks it's part of the selector.

I updated it to use braces like {uniquifier} for all these.